courseraresearchexports/commands/db.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Coursera
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from tabulate import tabulate
+
+import courseraresearchexports.db.db as db
+import courseraresearchexports.utils.container_utils as container_utils
+
+
+def list_tables(args):
+    """
+    List all of the tables present in a dockerized database.
+    """
+    d = container_utils.docker_client(args.docker_url, args.timeout)
+    tables = db.get_table_names(args.container_name_or_id, docker_client=d)
+    logging.info('\n' + tabulate([[table] for table in tables]))
+
+
+def list_views(args):
+    """
+    List all of the views present in a dockerized database.
+    """
+    d = container_utils.docker_client(args.docker_url, args.timeout)
+    tables = db.get_view_names(args.container_name_or_id, docker_client=d)
+    logging.info('\n' + tabulate([[table] for table in tables]))
+
+
+def create_view(args):
+    """
+    Create a view from a sql query.
+    """
+    d = container_utils.docker_client(args.docker_url, args.timeout)
+
+    if args.view_name:
```

Comment:
I think the `db` API should not know about whether the view came from a file or is 'pre-registered'. Can we factor that responsibility out elsewhere?